onadata/libs/utils/viewer_tools.py

```diff
@@ -170,12 +170,13 @@ def enketo_url(form_url,
                **kwargs):
     """Return Enketo webform URL."""
```

Comment:
We should probably update this functions description to clearly state that it now returns all the enketo urls. _The current descriptions still seems to point to the fact that this returns 1 url..... Maybe we should also change the name ?_

_We could also use type hints 😄 to show that this function should return a Dict/List_

Reply:
Cool, I can update the function name and description That must have slipped me.